cmd/vcpctl/provision/provision.go

```diff
@@ -94,6 +94,13 @@ func RunProvision(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "error: %v\n", err)
 		os.Exit(1)
 	}
+	defer client.Logout(ctx)
 	o.Client = client
+	fmt.Println("Create solution user...")
 	cli.CreateSolutionUser(ctx, &o)
+	r := cli.Role{RoleName: "default"}
```

Comment:
The role name should probably have a `k8s-vcp-` prefix or similar, so it is easier to identify from the UI, etc.

Reply:
I am thinking if we only need to create a "k8s-vcp-default" role with limited permissions in vcpctl. Do we need feature to add more roles or update role by vcpctl in the future ?

Reply:
Yes one role should be fine for now.  It may need to be updated in the future, if vcp expands the method set it currently uses.